larg4/Services/LArG4Detector_service.cc

```diff
@@ -164,20 +177,23 @@ std::vector<G4LogicalVolume *> larg4::LArG4DetectorService::doBuildLVs() {
             }
 
             if ((*vit).type == "StepLimit") {
-                G4UserLimits *fStepLimit = NULL;
+                G4UserLimits *fStepLimit = new G4UserLimits();
+                G4AutoDelete::Register(fStepLimit);
```

Comment:
See other comment about registration of pointers.  Any reason a stack variable would not work instead?

Reply:
This seems to work fine here.